### PR TITLE
Bug#148

### PR DESCRIPTION
**Pull request by Setnea**
Fixing a bug, removing mpld3 library.

### CODE REVIEW

File: mothulity/utilities.py [hunk 1]
```diff
@@ -522,16 +522,17 @@ def draw_rarefaction(input_file_name,
...
     plt.grid(True)
     plt.title(title)
     plt.ylabel(ylabel)
     plt.xlabel(xlabel)
+    # Embed the result in the html output.
+    buf = BytesIO()
+    plt.savefig(buf, format="png")
+    encoded = base64.b64encode(buf.getvalue()).decode('utf-8')
+    html ='<img src=\'data:image/png;base64,{}\'>'.format(encoded)
     with open(output_file_name, "wb") as fout:
```
Comment (dizak) [inline mothulity/utilities.py]: Change mode to ```w```

File: mothulity/utilities.py [hunk 2]
```diff
@@ -650,9 +651,13 @@ def draw_scatter(input_file_name,
...
     labels = list(df[group_col])
-    tooltip = mpld3.plugins.PointLabelTooltip(scatter, labels=labels)
-    mpld3.plugins.connect(fig, tooltip)
-    mpld3.save_html(fig, output_file_name)
+    # Embed the result in the html output.
+    buf = BytesIO()
+    plt.savefig(buf, format="png")
+    encoded = base64.b64encode(buf.getvalue()).decode('utf-8')
+    html ='<img src=\'data:image/png;base64,{}\'>'.format(encoded)
+    with open(output_file_name, "wb") as fout:
```
Comment (dizak) [inline mothulity/utilities.py]: Change mode to ```w```

Comment (Setnea) [reply mothulity/utilities.py]: Alright, totally forgot about it!

**Comment by Setnea** [comment]
I added a legend. I'm afraid that too many groups might make it look a little messy, but it is how it is.
I also changed the code in a way that it can be easily reused for plotting variants/groups of samples.
Have a nice weekend!

File: requirements.txt [hunk 3]
```diff
@@ -2,7 +2,6 @@ requests
 tqdm
 biopython
 matplotlib<=2.0.0
```
Comment (dizak) [inline requirements.txt]: Remove version requirement

File: requirements.txt [hunk 4]
```diff
@@ -2,7 +2,6 @@ requests
...
-mpld3
 numpy<=1.13.3
```
Comment (dizak) [inline requirements.txt]: Remove version requirement

**Comment by dizak** [comment]
@Setnea I see you are trying to solve why travis builds fails unexpectedly. Can you try commenting out the ```total``` argument?

https://github.com/dizak/mothulity/blob/c16907efc6f46aabae449a27287763ca786eda6d/mothulity/utilities.py#L443

**Comment by dizak** [comment]
```ascii``` argument in the ```tqdm``` function can accept ```bool``` or ```str```. Can you please try with passing ```ascii=' 123456789#'```? Yes, whitespace is the first char in this ```str```

**Pull request merged**